tests/integration/abi_roundtrip_test.py

```diff
@@ -101,6 +117,7 @@ def roundtrip_setup(abi_type):
 
     return (
         abi_type,
+        str(abi.type_spec_from_annotation(abi_type)),
```

Comment:
@tzaffi added this to the return tuple rather than writing another method to just get the string after doing the tuple check